app/wizards/placements/add_placement_wizard/preview_placement_step.rb

```diff
@@ -0,0 +1,23 @@
+class Placements::AddPlacementWizard::PreviewPlacementStep < Placements::BaseStep
+  delegate :school, to: :wizard
+  delegate :steps, to: :wizard
+
+  def placement
+    placement = school.placements.build
+    placement.subject = steps[:subject].subject
+
+    if steps[:additional_subjects].present?
+      placement.additional_subjects = steps[:additional_subjects].additional_subjects
+    end
+
+    if steps[:year_group].present?
+      placement.year_group = steps[:year_group].year_group
+    end
+
+    if steps[:mentors].present?
+      placement.mentors = steps[:mentors].mentors
+    end
+
+    placement
+  end
```

Comment:
Let's refactor this so we only build the placement _once_ in this entire wizard.

e.g.

- Break up `Placements::AddPlacementWizard#create_placement` into two methods:
- One to build a placement
- The other one to take the placement & save/create it.